.travis.yml

```diff
@@ -0,0 +1,24 @@
+dist: trusty
```

Comment:
Is it necessary to specify a `dist` for any of our production containers?

Reply:
Not necessary. Completely optional. If you want to stick with the same I can remove that line.